CalculatorTests/CalculatorTests/Tests/SettingsPageTests.cs

```diff
@@ -0,0 +1,180 @@
+﻿using System;
+using System.Collections.Generic;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Support.UI;
+
+namespace CalculatorTests.Tests
+{
+    [TestFixture]
+    public class SettingsPageTests
+    {
+        private IWebDriver driver;
+
+        [SetUp]
+        public void SetUp()
+        {
+            var options = new ChromeOptions { AcceptInsecureCertificates = true };
+            driver = new ChromeDriver(options);
+            driver.Url = "https://localhost:5001/Calculator";
+
+
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            driver.Quit();
+        }
+
+       [TestCase("dd/MM/yyyy", "123,456,789.00", "$ - US dollar", "$", "100,000.00", "200,000.00", "19/05/2023")]
+       [TestCase("dd-MM-yyyy", "123.456.789,00", "€ - euro", "€", "100.000,00", "200.000,00", "19-05-2023")]
+       [TestCase("MM/dd/yyyy", "123 456 789.00", "£ - Great Britain Pound", "£", "100 000.00", "200 000.00", "05/19/2023")]
+       [TestCase("MM dd yyyy", "123 456 789,00", "₴ - Ukrainian hryvnia", "₴", "100 000,00", "200 000,00", "05 19 2023")]
+
+
+        public void SettingsTests (string dateFormat, string numberFormat, string defaultCurrency, string amountCurrency, string interestEarned, string income, string endDate)
+        {
+            IWebElement settingsButton = driver.FindElement(By.XPath("//div[@class = 'settings link btn btn-link']"));
```

Comment:
You can avoid using the full class name:
```suggestion
            IWebElement settingsButton = driver.FindElement(By.XPath("//div[contains(@class, 'settings') ]"));
```

Reply:
Fixed